JavaScript/JavaScriptSDK/Context/User.ts

```diff
@@ -29,12 +33,60 @@ module Microsoft.ApplicationInsights.Context {
          * The store region.
          */
         public storeRegion: string;
-        
+
+        /**
+         * Authorized user id
+         */
+        private authId: string;
+        public getAuthId() {
```

Comment:
As a person reading this API for the first time, I find it confusing. What does `AuthId` mean? Is this an authentication token, an authorization token or something else? With JavaScript's support for doc comments and IntelliSense being very limited this method would also look confusing at the point of call and require users to refer to the API documentation. 

Can we spell this out as `AuthenticatedUserId`?